llvm/tools/opt/optdriver.cpp

```diff
@@ -630,7 +630,7 @@ extern "C" int optMain(
     }
   }
 
-  Triple ModuleTriple(M->getTargetTriple());
+  Triple ModuleTriple(Triple::normalize(M->getTargetTriple()));
```

Comment:
Similarly to my concern about updating `createTargetMachineForTriple`, I don't think opt should messing with the triple it's given.

Reply:
> Similarly to my concern about updating `createTargetMachineForTriple`, I don't think opt should messing with the triple it's given.

Deleted call to `normalize()`.